c2corg_ui/static/js/documentediting.js

```diff
@@ -151,6 +151,12 @@ app.DocumentEditingController = function($scope, $element, $attrs, appLang,
   this.submit = false;
 
   /**
+   * @type {Date}
+   * @export
+   */
+  this.today = new Date();
```

Comment:
`Today` is also used for outings, but it has been moved to the `outingEditingController`. So maybe I should remove it from there and leave only `today` in `documentEditingController`?